SynthSeg/segmentation_model.py

```diff
@@ -0,0 +1,41 @@
+from tensorflow import keras
+from typing import Tuple
+
+
+def unet(input_shape: Tuple[int, ...], num_classes):
```

Comment:
I think you mentioned that you wanted to re-implement the UNet. However, this doesn't seem to support all the training options like convolution size yet, right? What's the plan for that?

Reply:
Yes, currently this is just a dummy to test that the distributed training works. I am working on a proper implementation, will be a follow-up PR to make this PR a bit lighter.